plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,14 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getTasks() {
```

Comment:
But at the same time I'm indifferent if we need to keep following the legacy for stronger reasons.